testing/test_package/lib/fake.dart

```diff
@@ -496,7 +496,7 @@ class BaseForDocComments {
   ///
   /// Reference to a top-level const in another library [incorrectDocReferenceFromEx]
   ///
-  /// Reference to prefixed-name from another lib [css.theOnlyThingInTheLibrary] xx
+  /// Reference to prefixed-name from another lib [example_prefixed.incorrectDocReferenceFromEx]
```

Comment:
Does css.theOnlyThingInTheLibrary still fail?

Reply:
That member was removed – but since this test was skipped, it didn't change output.

Just picked something that already existed.

Reply:
The member isn't actually gone, so this test should pass (assuming we didn't have the bug it is testing for).